exercises_on_code_quality/ReverseVowelsInAString.js

```diff
@@ -0,0 +1,10 @@
+const reverseVowels = (str) => {
+  //The regex help to test or match all vowels in the string with out considering upper or lower cases.
+  let vowels = str.match(/[aiuoe]/gi);
+  return str
+    .split("")
+    .map((character) =>
+      /[aiuoe]/gi.test(character) ? vowels.pop() : character
```

Comment:
I would suggest you to add this regex **/[aiuoe]/gi** into its own variable so that it can be more clear and readable